Documentation/doc/Documentation/Developer_manual/Chapter_portability.txt

```diff
@@ -304,7 +304,7 @@ up-to-date version of this list.
 \subsection secworkaround_macros Macros connected to workarounds/compilers
 
 Some macros are defined according to certain workaround flags. This is
-done to avoid some \#`ifdef`s in our actual code.
+done to avoid some `#ifdef`s in our actual code.
```

Comment:
This results in a warning:
`Documentation/Developer_manual/Chapter_portability.txt:307: warning: explicit link request to 'ifdef' could not be resolved`

Reply:
Strange, with which version of doxygen?
When I run nmake Documentation_internal_doxygen_run or nmake doc with the current doxygen master (1.9.5 (86fe8ce908bfd6782633d2d9647f2159c2f08a44)) I don't get a warning.
With the overnight build [CGAL-5.5-Ic-80](https://cgal.geometryfactory.com/CGAL/Manual_doxygen_test/CGAL-5.5-Ic-80/index.html) I also see no indication of it.

Reply:
I did a small text with just a small text and used
```
some `#``ifdef`s in
```
and with the small test the 1.8.4, 1.8.13 and the current master give no warning and the output looks OK
(the internal code will look like  `some <code>#</code><code>ifdef</code>s in`, which is not nice but is OK as well).

When you see the problem with the 1.8.4 or the 1.8.13 version, we might use this version of the code.
(Though this is also a nice reminder of going to a newer version of doxygen  #5840)